lib/src/chewie_player.dart

```diff
@@ -42,6 +42,8 @@ class Chewie extends StatefulWidget {
 
 class ChewieState extends State<Chewie> {
   bool _isFullScreen = false;
+
+  bool get isFullScreen => widget.controller.isFullScreen;
```

Comment:
Might be better to rename this to `isControllerFullScreen` so that it's distinct from `_isFullScreen`.